### PR TITLE
Apps.ListRepos/Apps.ListUserRepos: Add mercy-preview + nebula-preview headers

### PR DESCRIPTION
**Pull request by andygrunwald**
**What type of PR is this?**

feature

**What this PR does / why we need it**:

* mercy-preview: For receiving repository topics
* nebula-preview: For visibility information

When calling `Apps.ListRepos` or `Apps.ListUserRepos`, the field `visibility` and `topic` are always `nil`.

The reason: Missing headers for developers preview API.

**Special notes for your reviewer**:

This applies the same fix as in https://github.com/google/go-github/pull/1780

**Does this PR introduce a user-facing change?**:

None

**Additional documentation e.g., usage docs, etc.**:

- [API changes to support internal repository visibility (2019-12-03)](https://developer.github.com/changes/2019-12-03-internal-visibility-changes/#repository-visibility-fields)

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @andygrunwald !
~I've gone through the GitHub v3 API docs and can't find any mention of the nebula-preview custom header... why is that one needed?~  EDIT: Ah!  I see you already covered this in your description... awesome.  Thank you!

Otherwise, LGTM.

Awaiting second LGTM before merging.

**Comment by gmlewis** [comment]
Never mind my last comment.  :joy:  See edit above.

**Review by wesleimp — APPROVED** [review]
💙

**Comment by andygrunwald** [comment]
@gmlewis @wesleimp Good to merge? Anything I can do to support?

**Comment by gmlewis** [comment]
Thank you, @wesleimp !
Merging.

**Pull request merged**
